docs/getting-started/installation.md

```diff
@@ -72,7 +72,7 @@ mamba create -n bespokefit-env -c conda-forge/label/libint_dev -c conda-forge py
 or with OpenEye Toolkits:
 
 ```shell
-mamba create -n bespokefit-env -c conda-forge-label/libint_dev -c conda-forge -c openeye python=3.10 openff-bespokefit "qcportal <0.50" psi4 openeye-toolkits
+mamba create -n bespokefit-env -c conda-forge/label/libint_dev -c conda-forge -c openeye python=3.10 openff-bespokefit "qcportal <0.50" psi4 openeye-toolkits
```

Comment:
Oof ... my bad.

This should also have `-c psi4 ... dftd3`

Reply:
I think I caught that one as you were typing the comment :-) Thanks for another set of eyes.